README.rst

```diff
@@ -222,7 +222,7 @@ PyTorch SageMaker Estimators
 
 With PyTorch SageMaker Estimators, you can train and host PyTorch models on Amazon SageMaker.
 
-Supported versions of PyTorch: ``0.4.0``, ``1.0.0``, ``1.1.0``.
+Supported versions of PyTorch: ``0.4.0``, ``1.0.0``, ``1.1.0``, ``1.2.0``.
 
 We recommend that you use the latest supported version, because that's where we focus most of our development efforts.
 
```

Comment:
also, the table at the bottom of this README should be updated (feel free to do in a separate PR)